SPECS-EXTENDED/python-cheetah/python-cheetah.spec

```diff
@@ -1,109 +1,142 @@
-%bcond_with python2
-
 Name:           python-cheetah
-Version:        3.2.4
-Release:        4%{?dist}
+Version:        3.2.6.post1
+Release:        12%{?dist}
 Summary:        Template engine and code generator
 
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://cheetahtemplate.org/
-Source:         https://github.com/CheetahTemplate3/cheetah3/archive/%{version}/%{name}-%{version}.tar.gz
-BuildRequires:  gcc
+Source:         https://github.com/CheetahTemplate3/cheetah3/archive/%{version}/Cheetah3-%{version}.tar.gz
+
+# Instead of playing Whac-A-Mole and adding more and more basepythons,
+# e.g. in https://github.com/CheetahTemplate3/cheetah3/commit/6be6bc10a4,
+# we let tox do the right thing by not setting any:
+Patch:          tox-no-basepython.patch
+Patch1:         cheetah3-3.2.6.post1-protect-cgi.patch
+Patch2:         cheetah3-3.2.6.post1-loadTestsFromModule.patch
+Patch3:         cheetah3-3.2.6.post1-typeerror.patch
+Patch4:         cheetah3-3.2.6.post1-framelocalsproxy.patch
+Patch5:         cheetah3-3.2.6.post1-parse_qs.patch
 
-%global _description\
-Cheetah is an open source template engine and code generation tool,\
-written in Python. It can be used standalone or combined with other\
-tools and frameworks. Web development is its principal use, but\
-Cheetah is very flexible and is also being used to generate C++ code,\
-Java, SQL, form emails and even Python code.
+BuildRequires:  gcc
+BuildRequires:  python3-devel
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+
+# for tests
```

Comment:
Are these dependencies required for running package tests? If yes, we need to move them under `%if %{with_check}` marco